src/FlightDisplay/FlightDisplayView.qml

```diff
@@ -728,6 +729,140 @@ Item {
+            id:     annunciatorRow
+            spacing:    ScreenTools.defaultFontPixelWidth
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.verticalCenter: parent.verticalCenter
+
+            Repeater {
+                id: annunciatorRepeater
+                readonly property double    ekfLowThresh:   0.5
+                readonly property double    ekfHighThresh:  0.8
+                readonly property double    vibeLowThresh:  30
+                readonly property double    vibeHighThresh: 50
+                readonly property double    tiltLowThresh:  15
+                readonly property double    tiltHighThresh: 30
+
+                function getColor(value, lowThresh, highThresh) {
+                    if(value > highThresh)      return qgcPal.colorRed
+                    else if(value > lowThresh)  return qgcPal.colorOrange
+                    else                        return qgcPal.colorGreen
+                }
+
+                function getEKFColor(value) {
+                    return getColor(value, ekfLowThresh, ekfHighThresh)
+                }
+
+                function getLargestOf(v1, v2, v3) {
+                    return Math.max(Math.abs(v1),Math.abs(v2),Math.abs(v3))
+                }
+
+                function getEKFVelColor() {
+                    if(!activeVehicle.estimatorStatus.goodHorizVelEstimate.value) return qgcPal.colorGrey
+                    return getEKFColor(activeVehicle.estimatorStatus.velRatio.value)
+                }
+
+                function getEKFHPOSColor() {
+                    if(!activeVehicle.estimatorStatus.goodHorizPosAbsEstimate.value) return qgcPal.colorGrey
+                    return getEKFColor(activeVehicle.estimatorStatus.horizPosRatio.value)
+                }
+
+                function getEKFVPOSColor() {
+                    if(!activeVehicle.estimatorStatus.goodVertPosAbsEstimate.value) return qgcPal.colorGrey
+                    return getEKFColor(activeVehicle.estimatorStatus.vertPosRatio.value)
+                }
+
+                function getEKFMAGColor() {
+                    return getEKFColor(activeVehicle.estimatorStatus.magRatio.value)
+                }
+
+                function getEKFTerrColor() {
+                    if(!activeVehicle.estimatorStatus.goodVertPosAGLEstimate.value) return qgcPal.colorGrey
+                    return getEKFColor(activeVehicle.estimatorStatus.haglRatio.value)
+                }
+
+                function getVibeColor() {
+                    return getColor(
+                                getLargestOf(
+                                    activeVehicle.vibration.xAxis.value,
+                                    activeVehicle.vibration.yAxis.value,
+                                    activeVehicle.vibration.zAxis.value),
+                                vibeLowThresh, vibeHighThresh)
+                }
+
+                function getTiltColor() {
+                    return getColor(
```

Comment:
Does it handle negative value ? (can tilt be negative?)

Reply:
`getAbsLargestOf` uses the absolute value of each.  I've renamed the method for clarity.